pytests/test_execution.py

```diff
@@ -215,3 +215,46 @@ def mapper(item):
 
         assert test_proc.exitcode == 99
         assert len(out) < 1000
+
+
+def test_yield_epochs_with_cluster_main(mp_ctx):
```

Comment:
is this more overhead than we need? Maybe. But this would have caught the missed signature change required.